samples/bot-formatting-cards/csharp/BotFormattingCards/Resources/stageViewForImages.json

```diff
@@ -0,0 +1,20 @@
+{
+    "type": "AdaptiveCard",
+    "body": [
+        {
+            "type": "TextBlock",
+            "text": "Stage View Images",
+            "size": "Large",
+            "weight": "Bolder"
+        },
+        {
+            "type": "Image",
+            "url": "https://picsum.photos/200/200?image=110",
```

Comment:
What is this URl?
https://picsum.photos/200/200?image=110
Can we use it or it's a copy right violation?

Reply:
As an example, I've chosen this image from learn.microsoft.com. 